Description: Move the “overall information” manual page to correct section.
 .
 The “entire documentation” manual page generated is not suitable for a section
 1 (“User commands”) page.
 .
 We move it to section 7 (“Overview”).
Bug-GitHub: https://github.com/twisted/towncrier/issues/469
Author: Ben Finney <bignose@debian.org>
Last-Update: 2023-01-06

diff --git old/docs/conf.py new/docs/conf.py
--- old/docs/conf.py
+++ new/docs/conf.py
@@ -133,7 +133,7 @@
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [(master_doc, "towncrier", "Towncrier Documentation", [author], 1)]
+man_pages = [(master_doc, "towncrier", "Towncrier", [author], 7)]
 
 
 # -- Options for Texinfo output -------------------------------------------


Local variables:
coding: utf-8
mode: diff
time-stamp-format: "%:y-%02m-%02d"
time-stamp-start: "^Last-Update:[ 	]+"
time-stamp-end: "$"
time-stamp-line-limit: 20
End:
vim: fileencoding=utf-8 filetype=diff :
